### ISSUE TITLE
Increase GPU's total session runtimes!

### ISSUE THREAD

**Issue by lorazabora** [aws/studio-lab-examples#121]
**Is your feature request related to a problem? Please describe.**
Yes, it's related to a lack of usage within the service

**Describe the solution you'd like**
I would like total session runtimes on GPU's to get back as they were or at least increase, since 8 hours is very low in my opinion, i know the service have had huge demand and issues recently but i'd to at least know if there will be future plans for brining the old system back if everything settles in!

**Describe alternatives you've considered**
Increasing total GPU runtime for at least 16-20 hours in a 24-hour period whenever GPU demand settles, i see that as a very good range of sessions

**Additional context**
None

**Comment by pixepchief** [comment]
Agreed!

**Comment by lorazabora** [comment]
> Agreed! 8 hours are definitely not enough as some projects tend to take more time usually than this. As aforementioned, you should bring back the old session timing for the GPUs and fix the "No runtime errors" which most of us faced recently.

yes, that's exactly what we'd all like to see, the service getting back to how it was before when it started, 4 hour session limits and guaranteed GPU capacity for everyone, in my opinion the focus should be more on increasing the capacity of them than adding more limitations

**Comment by pero03** [comment]
i agree as well

this is very unfair

some of my work used to take a very long time but i was still able to do them because there was good gpus

i say bring back the old session system

**Comment by nzy0939** [comment]
> yes kindly remove that limit even if gpus are not guaranteed it's still much better

i don't mind that too

**Comment by datanetai** [comment]
I think it's okay to have limitation on free service. Otherwise some people will keep GPU on then it will not be available for other. Also having 8 hour per day is good option. I can do lot of thing in 8 hours.

**Comment by lorazabora** [comment]
> I think it's okay to have limitation on free service. Otherwise some people will keep GPU on then it will not be available for other. Also having 8 hour per day is good option. I can do lot of thing in 8 hours.

We all know it's a free service, but i disagree with you

first of all it's a locked service meaning they get to choose who hops in and who don't, second of all no people cannot keep GPU's on anyways because there's already the 4 hour session limit

and last thing is that you say you can do alot of things with it, well that's you which is personal you don't get to decide the same thing for all users

having it back as before is what suits everyone and that's why the focus should be on increasing the capacity instead of such a limit

i hope that's clear, have a great rest of your day! :)

**Comment by Penixlast14** [comment]
> 

i agree!
i also believe that we would rather have the gpus not being available to us than having a 4 hour limit per 12 hours and 8 hours per 24 hours period, having gpus not being available is much much better than this i would rather have that, please sagemaker don't let us down, you've been a really good and helpful service to all of us! :)

**Comment by lorazabora** [comment]
> > 
> 
> i agree! i also believe that we would rather have the gpus not being available to us than having a 4 hour limit per 12 hours and 8 hours per 24 hours period, having gpus not being available is much much better than this i would rather have that, please sagemaker don't let us down, you've been a really good and helpful service to all of us! :)

exactly, as i said again in my opinion the focus should be more on brining the service back to how it was back in 2021
whereas capacity is within control and you always find a GPU + no total limitation the regular 4 hour session limit is good enough

**Comment by Penixlast14** [comment]
> 

mhm! in my opinion i think all of us can agree that we'd rather have gpus not being guaranteed and having the "gpu not available" error much much more than these limitations, sagemaker studio lab please hear our pleas!!!

**Comment by Penixlast14** [comment]
i don't believe they want to do it

**Comment by lorazabora** [comment]
> i don't believe they want to do it

just stay patient man, it's not as easy as you might think

we all want it to improve but give them some time

**Comment by lorazabora** [comment]
> i don't believe they want to do it

in the meantime they probably have to keep it with this limitation, but hopefully within the next month or two it changes again to the better

**Comment by 03Sys** [comment]
One year later, it's now 4 hours in total.

**Comment by kwekewk** [comment]
> One year later, it's now 4 hours in total.

and reduced cpu runtime.
